File: Frontend/src/modules/inventories/interfaces/CreateInventoryDto.ts
export interface CreateCustomFieldDto {
  name: string;
  type: string;
  showInTable: boolean;
  sortOrder: number;
  validationRules?: string;
  options?: string;
}
export interface CreateInventoryDto {
  title: string;
  description?: string;
  categoryName: string;
  isPublic: boolean;
  tags?: string[];
  ownerId: string;
  customFields?: CreateCustomFieldDto[];
}

File: Backend/ToryBack/Models/DTOs/CreateInventoryDto.cs
namespace ToryBack.Models.DTOs
{
    public class CreateInventoryDto
    {
        public string Title { get; set; } = string.Empty;
        public string? Description { get; set; }
        public string CategoryName { get; set; } = string.Empty;
        public bool IsPublic { get; set; }
        public string OwnerId { get; set; } = string.Empty;
        public List<string>? Tags { get; set; }
        public List<CreateCustomFieldDto>? CustomFields { get; set; }
    }
}
